contracts/tasks/deploy-gateway.ts

```diff
@@ -0,0 +1,92 @@
+import { task, types } from 'hardhat/config'
+import { HardhatRuntimeEnvironment, TaskArguments } from 'hardhat/types'
+import { Deployments } from './lib'
+
+// Customize your gateway parameters here.
+const gatewayConstructorParams = {
+    bottomUpCheckPeriod: 10,
+    activeValidatorsLimit: 100,
+    majorityPercentage: 66,
+    networkName: {
+        root: undefined, // Will be set later.
```

Comment:
As in the program will auto set it for you?

Reply:
It's done below.